zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -36,6 +36,9 @@
     private static final String IMAGE_API_URL = "http://localhost:24700";
     private static final String KAFKA_ADDR = "localhost:9092";
     private static final String KAFKA_CONTENT_PUBLISHED_TOPIC = "content-published";
+    private static final String KAFKA_SEC_PROTO = "";
+    private static final String KAFKA_SEC_CLIENT_KEY_P12 = "";
+    private static final String KAFKA_SEC_CLIENT_KEY = "";
```

Comment:
Why is the default protocol blank? If a default isn't needed we don't need the static variable. Similarly the key values do not need to be defined if they are blank.